packages/generator/templates/form/__ModelName__Form.tsx

```diff
@@ -1,13 +1,22 @@
+import {Suspense} from "react"
 import {Form, FormProps} from "src/core/components/Form"
 import {LabeledTextField} from "src/core/components/LabeledTextField"
+
 import {z} from "zod"
 export {FORM_ERROR} from "src/core/components/Form"
 
-export function __ModelName__Form<S extends z.ZodType<any, any>>(props: FormProps<S>) {
+function __ModelName__FormSuspense<S extends z.ZodType<any, any>>(props: FormProps<S>) {
   return (
     <Form<S> {...props}>
-      <LabeledTextField name="name" label="Name" placeholder="Name" />
+      {/* template: <__component__ name=__fieldName__ label=__Field_Name__ placeholder=__Field_Name__  type=__inputType__ /> */}
```

Comment:
I think we should remove the quotes from the generated values and add them in the template if they are needed. Otherwise it's guaranteed we'll run into place where we don't actually want the quotes in the value.

```suggestion
      {/* template: <__component__ name="__fieldName__" label="__Field_Name__" placeholder="__Field_Name__"  type="__inputType__" /> */}
```